media/transcoder/job_test.py

```diff
@@ -252,3 +357,11 @@ def _get_job_state(capsys, job_id):
 
     out, _ = capsys.readouterr()
     assert job_succeeded_state in out
+
+
+def _get_file_in_bucket(capsys, directory_and_filename):
+    storage_client = storage.Client()
```

Comment:
We shouldn't be creating a new client like this - please just reuse the same client. You can create a fixture for it and scope to the module to avoid having to recreate it several times.

Reply:
Refactored to use the test bucket